packages/extension-mention/src/mention.ts

```diff
@@ -9,6 +9,7 @@ export type MentionOptions = {
     options: MentionOptions,
     node: ProseMirrorNode,
   }) => string,
+  deleteOnBackspace: boolean,
```

Comment:
I want to add my thoughts to the naming, feels a bit too abstract. The default behaviour is already "delete on backspace", this option doesn't really clarify what it actually does. People reading the documentation might be confused and think this needs to be always `true` otherwise backspace might not work, which is not true.

My suggestion, rename all references to: `deleteTriggerCharWithBackspace` or something along those lines, something more obvious, more explicit...

Reply:
Sounds good! The option is a bit long. Do you think `deleteTriggerWithBackspace` would also be fitting? Is it clear that the trigger is the actual trigger char?

Usually I like to go with shorter, easier to remember options (even though Typescript should boot this argument out of the window).

Maybe even go with the full word if we don't care about option length? `deleteTriggerCharacterWithBackspace` or `includeTriggerCharacterOnDeletion`?

Reply:
Whatever you're most comfortable, I just think the original one is a bit misleading.

Reply:
@Tommy-Sun could we go with `deleteTriggerWithBackspace`?

Reply:
@bdbch added 👍